### PR TITLE
SOE-811: added banner_850x400

### PR DESCRIPTION
**Pull request by cjwest**
@boznik
Here's the second of several PRs for the news feature

### CODE REVIEW

**Comment by cjwest** [comment]
@boznik 

Ready for your review!

File: stanford_image.strongarm.inc [hunk 1]
```diff
@@ -175,6 +175,18 @@ function stanford_image_strongarm() {
       'stanford_huge_landscape' => array(
         'custom_settings' => FALSE,
       ),
+      'stanford_banner_850x400' => array(
+        'custom_settings' => TRUE,
+      ),
+      'gallery_large' => array(
```
Comment (sherakama) [inline stanford_image.strongarm.inc:181]: @cjwest @boznik - These gallery strongarm settings cannot be in here. They belong in the stanford_gallery module. Please remove them.

Comment (cjwest) [inline stanford_image.strongarm.inc:181]: So these lines need to be removed?

```
  'gallery_large' => array(
    'custom_settings' => FALSE,
  ),
  'gallery_medium' => array(
    'custom_settings' => FALSE,
  ),
  'gallery_small' => array(
    'custom_settings' => FALSE,
  ),
```

Comment (sherakama) [inline stanford_image.strongarm.inc:181]: yes.

Comment (cjwest) [inline stanford_image.strongarm.inc:181]: Since this branch was deleted, I put it here:
https://github.com/SU-SWS/stanford_image/pull/38

**Pull request merged**